src/plugins/kibana_utils/common/persistable_state/types.ts

```diff
@@ -106,9 +106,6 @@ export type PersistableStateMigrateFn = (
   version: string
 ) => SerializableRecord;
 
-/**
- * @todo Shall we remove this?
- */
```

Comment:
I don't think we should remove this, at least not due to lack of usage. Was there another reason we didn't want to keep this type around?

Reply:
cc @vadimkibana

Reply:
LGTM to remove this comment - I don't see any context